sub/sd_ass.c

```diff
@@ -475,8 +475,10 @@ static void configure_ass(struct sd *sd, struct mp_osd_res *dim,
             // ffmpeg and mpv use a default PlayResX of 384 when it is not known,
             // this comes from VSFilter.
             double fix_margins = track->PlayResX / (double)MP_ASS_FONT_PLAYRESX;
```

Comment:
Now, if you could also copy the old, unscaled `PlayResX` tow lines prior, and then replace `MP_ASS_FONT_PLAYRESX` in this line with `old_playresx` instead of wrongly assuming the old value always was `384=MP_ASS_FONT_PLAYRESX`, this would be perfect.

Reply:
Thanks! I intended to do this but it got lost in the discussion last time around...